app/actions/EventActions.ts

```diff
@@ -418,3 +419,13 @@ export function isUserFollowing(eventId: number): Thunk<any> {
     },
   });
 }
+export function fetchAnalytics(eventId: ID): Thunk<Promise<Action>> {
```

Comment:
Since you're using the data from this directly, it would be nice if you could type this properly :)

Reply:
I typed it now, but to do so I had to make a bunch of changes to the call api. Take a look.

Reply:
I removed the typing commit since it didn't work and got out of scope here. You can now find it on branch `type-actions`.